content/en/docs/setup/_index.md

```diff
@@ -102,6 +97,3 @@ When you're ready to consider more advanced Istio use cases, check out the follo
 - To expand your existing mesh with additional containers or VMs not running on
 your mesh's Kubernetes cluster, follow our [mesh expansion guide](/docs/examples/mesh-expansion/).
 
-- To add services requires a detailed understanding of sidecar injection. Visit our
```

Comment:
I agree that the wording can be improved but please keep it on the list. The purpose of the list is to point readers to more scenarios after setup.
```
- To add services using sidecar injection, see our

Reply:
Thanks, I updated